### ISSUE TITLE
 high contrast theme can not be changed anymore

### ISSUE THREAD

**Issue by Diyorbek7747** [microsoft/vscode#125770]
Hello,

After some uses of Visual Code in normal mode, I've set the high contrast functionality and the code changes for HC colors. After sometime, when back to normal mode, the high contrast color still on VSCode and cannot be changed anymore to the normal configs. I've tried to restart the computer, but doesn't change anything.My PC's OS is Windows 10.

Thank you!

**Comment by aeschli** [comment]
You are saying the theme can not be changed when you go to `Preference: Color Theme`?
Or is it that the high contrast theme comes back after the next restart?

**Comment by aeschli** [comment]
Here are the steps I tried and they worked for me:
- VSCode open
- In the WIndows 10 settings, turn on High Contrast
- verify that VSCode also has switched to High contrast. This is because of the setting `window.autoDetectHighContrast`
- also verify that the theme setting is now `"workbench.colorTheme": "Default High Contrast"`
- while VSCode is still running, In the WIndows 10 settings, turn off High Contrast
- verify that VSCode is now back to the theme as before

If VSCode is closed while the WIndows 10 settings turns off High Contrast, then VSCode that comes up will still be in high contrast.
You can the go to `Preference: Color Theme` and manually change the theme.

**Comment by aeschli** [comment]
@Diyorbek7747 an you give detailed steps on what you do and describe what you see and what you expect.

**Comment by isabelacmor** [comment]
I'm also seeing this issue. I enabled WHCM in settings and VSCode updated to reflect HC. Then I disabled WHCM and VSCode still shows up in HC. Restarting VSCode doesn't update the theme. I have to manually change my theme in Preferences.

Every other program reverts back to the "normal" theme when WHCM is turned off and it feels like the expected behaviour for VSCode should follow the same pattern.
